app/javascript/users/UserEdit.re

```diff
@@ -0,0 +1,584 @@
+                active={hasInvalidPassword(state)}
+              />
+            </div>
+          </div>
+        </div>
+        <div className="flex flex-col md:flex-row mt-10 md:mt-12">
+          <div className="w-full md:w-1/3 pr-4">
+            <h3 className="text-lg font-semibold">
+              {"Notifications" |> str}
+            </h3>
+            <p className="mt-1 text-sm text-gray-700">
+              {"Update settings for email notifications." |> str}
+            </p>
+          </div>
+          <div className="mt-5 md:mt-0 w-full md:w-2/3">
+            <p className="font-semibold"> {"Community Digest" |> str} </p>
+            <p className="text-sm text-gray-700">
+              {"Community digest emails contain new questions from your communities, and a selection of unanswered questions from the past week."
+               |> str}
+            </p>
+            <div className="mt-6">
+              <div className="flex items-center">
+                <Radio
+                  id="daily_mail_enable"
+                  label="Send me a daily email"
+                  onChange={event =>
+                    send(
+                      UpdateDailyDigest(
+                        ReactEvent.Form.target(event)##checked,
+                      ),
+                    )
+                  }
+                  checked={state.dailyDigest}
+                />
+              </div>
+              <div className="mt-4 flex items-center">
+                <Radio
+                  id="daily_mail_disable"
+                  label="Disable"
+                  onChange={event =>
+                    send(
+                      UpdateDailyDigest(
+                        !ReactEvent.Form.target(event)##checked,
+                      ),
+                    )
+                  }
+                  checked={!state.dailyDigest}
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        className="bg-gray-100 px-4 py-5 sm:p-6 flex rounded-b-lg justify-end">
+        <button
+          disabled={saveDisabled(state)}
+          onClick={updateUser(state, send, currentUserId)}
+          className="btn btn-primary">
+          {"Save Changes" |> str}
+        </button>
+      </div>
+    </div>
+    <div className="bg-white shadow sm:rounded-lg mt-10">
+      <div className="px-4 py-5 sm:p-6">
+        <div className="flex flex-col md:flex-row">
+          <div className="w-full md:w-1/3 pr-4">
+            <h3 className="text-lg font-semibold"> {"Account" |> str} </h3>
+            <p className="mt-1 text-sm text-gray-700">
+              {"Manage your account in this school" |> str}
+            </p>
+          </div>
+          <div className="mt-5 md:mt-0 w-full md:w-2/3">
+            <p className="font-semibold text-red-700">
+              {"Delete account" |> str}
+            </p>
```

Comment:
There's no reason to request deletion again until the current request has expired.

Right now, it looks like the request can be repeatedly made, which means a user can trigger infinite deletion emails... this needs both a server-side block / check, and a client-side change to inform the user to check their inbox.